### ISSUE TITLE
Stuck on Initializing Auth Module

### ISSUE THREAD

**Issue by Shockshwat** [AdguardTeam/AdGuardHome#6688]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not [ask a question or ask for help](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a)

- [X] I have set up AdGuard Home correctly and [configured clients to use it](https://github.com/AdguardTeam/AdGuardHome/wiki/Clients). (Use the [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) for help with installing and configuring clients.)

### Platform (OS and CPU architecture)

Windows, AMD64 (aka x86_64)

### Installation

GitHub releases or script from README

### Setup

On one machine

### AdGuard Home version

0.107.73

### Action

```
AdGuardHome.exe
```

### Expected result

Open a local webpage with setup

### Actual result

Stuck on initializing auth module

### Additional information and/or screenshots

![image](https://github.com/AdguardTeam/AdGuardHome/assets/71540383/cd2fd0c8-f97a-4d1b-b1ff-51b8a6fc0f86)

**Comment by ainar-g** [comment]
Do you see anything on `127.0.0.1:3000`?  What are the messages when you launch AGH with `-v`?

**Comment by tjharman** [comment]
And what happens if you click on "Get Started" ?

**Comment by Shockshwat** [comment]
@tjharman It goes through the setup, However i dont think it works since it doesnt give me a DNS address to put into the router and devices.

**Comment by ainar-g** [comment]
@Shockshwat, if you want your devices to use it, you should probably install it onto the router itself or some other machine that isn't turned off and doesn't hibernate.

As for whether it works or not, if you see the dashboard in the UI and can reach it using `nslookup.exe` with any domain and one of the IPs listed on the *Setup guide* page.

**Comment by Shockshwat** [comment]
@ainar-g how does one install it onto the router itself? I only got some files which i do not know how to install on the router, its also a generic one i believe.

**Comment by john-dewey** [comment]
I had a similar issue. I was convinced that maybe Windows Firewall Defender was restricting access to Port 3000, but I found out that the path to the AdGuardHome.exe (which was on my desktop) had OneDrive get in the way. So, I moved the folder into my User directory (C:\user\AdGuardHome). This was enough to effectively get me out of this endless "Initializing auth module" step. Try moving it out of downloads to your user directory and see if that yields any results.
